syntax = "proto3"; // Set to proto2 if you are using Protocol Buffer Version 2

package arithmetic;

message AverageRequest
{
    int32 number = 1;
}

message AverageResponse
{
    float answer = 1;
}

service AverageService
{
    rpc Average(stream AverageRequest) returns (AverageResponse){};
}
